cmd/blob-server/main.go

```diff
@@ -156,6 +158,49 @@ func main() {
 		})
 	})
 
+	router.GET("/avail/verify/:blockHeight/:extIndex", func(c *gin.Context) {
+		blockHeight := c.Param("blockHeight")
+		extIndex := c.Param("extIndex")
+		blockHeightUint, err := strconv.ParseUint(blockHeight, 10, 64)
+		if err != nil {
+			c.JSON(http.StatusBadRequest, gin.H{
+				"message": fmt.Sprintf("blockHeight %s is not a valid uint64", blockHeight),
+			})
+			return
+		}
+		extIndexUint, err := strconv.ParseUint(extIndex, 10, 64)
+		if err != nil {
+			c.JSON(http.StatusBadRequest, gin.H{
+				"message": fmt.Sprintf("extIndex %s is not a valid uint64", extIndex),
+			})
+			return
+		}
+		verifier, err := availVerifier.NewDAVerifier(
+			"./avail-config.json",
+			ethEndpoint,
+			"0x967F7DdC4ec508462231849AE81eeaa68Ad01389", // Avail bridge deployed on Sepolia
+			"0x6B26173C8afF316919542df8dA5A57888e398ee1", // Custom Vector verifier contract deployed on Sepolia
```

Comment:
for these values can we have a constants file, which keeps values in this way.
```
{ [ENV]: { [DA]: {} } } 
```

Reply:
Like a `json` file? We'll have to parse it whenever`blob-server` runs.
A simple `constants.go`should be simple enough know?